packages/typescript-typings/types/ethers/index.d.ts

```diff
@@ -13,6 +13,7 @@ declare class Web3Provider {
   getBalance(address: string): Promise<BigNumber>;
   on(event: string, callback: (...args) => void): void;
   removeAllListeners(event: string): void;
+  estimateGas(tx): Promise<any>;
```

Comment:
I'd just remove this entire folder altogether, its outdated

Reply:
@snario It's not relevant at this moment, and also until Ethers support ES modules (ethers-io/ethers.js#114) we can't import it properly into Stencil.

Reply:
Well apparently it's still being used...

cc @joelalejandro